src/DIRAC/DataManagementSystem/DB/FileCatalogComponents/DirectoryManager/DirectoryTreeBase.py

```diff
@@ -723,9 +723,15 @@ def getDirectoryReplicas(self, lfns, allStatus=False):
 
     return S_OK({'Successful': successful, 'Failed': failed})
 
-  def getDirectorySize(self, lfns, longOutput=False, rawFileTables=False):
-    """ Get the total size of the requested directories. If long flag
+  def getDirectorySize(self, lfns, longOutput=False, rawFileTables=False, recursiveSum=True):
+    """
+        Get the total size of the requested directories. If long flag
         is True, get also physical size per Storage Element
+
+        :param longOutput: if True, also fetch the physical size per SE
+        :param rawFileTables: if True, use the File table instead of the precomputed values
+        :param recursiveSum: if True (default), takes into account subdirectories
```

Comment:
```suggestion
        :param bool longOutput: if True, also fetches the physical size per SE
        :param bool rawFileTables: if True, uses the File table instead of the pre-computed values
        :param bool recursiveSum: if True (default), takes into account subdirectories
```